README.md

```diff
@@ -1,7 +1,7 @@
 # OSS LLM Inference Template
 
 ## License
-This template is licensed to Customer subject to the terms of the license agreement between Domino and the Customer on file.
+This template is licensed under Apache 2.0 and contains the following open source components: Transformer (Apache 2.0)[https://github.com/huggingface/transformers/blob/main/LICENSE], Falcon (Apache 2.0)[https://huggingface.co/tiiuae/falcon-7b/blob/main/README.md], MLFlow (Apache 2.0)[https://github.com/mlflow/mlflow/blob/master/LICENSE.txt], NVIDIA [EULA](https://docs.nvidia.com/cuda/eula/index.html#license-grant)).
```

Comment:
the parenthesis don't look balanced: 
<img width="1544" alt="Screen Shot 2023-12-20 at 1 25 12 PM" src="https://github.com/dominodatalab/reference-project-llm-inference/assets/6508755/ba2fd514-8e09-4c7c-9600-dab3ee5bfec6">